requirements.txt

```diff
@@ -1,3 +1,4 @@
+antlr4-python3-runtime==4.7.2
```

Comment:
The reason for this is e.g. Travis installing `antlr` version 4.8, but SF is only compatible with `antlr` 4.7.3 (creates a problem for the integration tests on Python 3.6).